sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -2148,6 +2148,21 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase with Compilat
       ))
   }
 
+  // scalastyle:off line.size.limit
+  def nonFoldableInputError(argumentName: String, expression: Expression, requiredType: DataType): Throwable = {
```

Comment:
Just put parameters on separate lines, see https://github.com/databricks/scala-style-guide?tab=readme-ov-file#spacing-and-indentation

Reply:
Very helpful resource. Thanks, Max.